pom.xml

```diff
@@ -363,13 +363,17 @@
                                     <filtering>true</filtering>
                                     <includes>
                                         <include>config/*.yml</include>
+                                        <include>matchminerengine/*.py</include>
```

Comment:
Shouldn't `matchminerengine/**/*.py` include `matchminerengine/*.py` already?

Reply:
You're right. I'll change it. Thanks!